src/com/collection/DynamicArrayForStudents.java

```diff
@@ -33,14 +33,14 @@ public void setCapacity(int capacity) {
     public DynamicArrayForStudents()
     {
         setCapacity(10);      //let's assume default capacity = 10
```

Comment:
You may create a constant for 10 and use it in code

Reply:
okay